### ISSUE TITLE
Update the colors according to a pallete

### ISSUE THREAD

**Issue by igrangel** [i40-Tools/AlligatorWorkbench#2]
The colors used in the application should be updated according to [this ](https://coolors.co/) pallette.

**Comment by igrangel** [comment]
And also according to a bootstrap template, e.g., https://mdbootstrap.com/freebies/, http://www.free-css.com/, https://startbootstrap.com/template-categories/all/.

**Comment by igrangel** [comment]
I vote for [this](https://mdbootstrap.com/download/templates/bootstrap/free/jumbotron/index.html), what do you think?

**Comment by omarrana** [comment]
hi irlan, bootstrap is already included. We just need to find the template we want to use.The one created is with bootstrap template. You can download the fronend and edit index.html to change the template.The one you pointed is going towards bootstrap website. Do you have any specific template?

**Comment by igrangel** [comment]
I know that bootstrap is already included. I'm asking for your opinion regarding this template. The link above is one free bootstrap template.

**Comment by omarrana** [comment]
yes looks good. Do you have an idea how the web should look like.? 
The best way is to download the template and have a look.
And you can also change the text. i can embed it later.
If you want to finalise this we can start editing it and then later we can add it to web app.

**Comment by igrangel** [comment]
Looks like for using the previous one one has to pay it. Therefore I would like something like (this)[https://blackrockdigital.github.io/startbootstrap-modern-business/services.html].

**Comment by igrangel** [comment]
https://coolors.co/export/png/86bbd8-d1cfe2-cc8b86-ecce8e-b4edd2

**Comment by LIUHaohui** [comment]
I think the color pallete looks good. better than the color of https://mdbootstrap.com/download/templates/bootstrap/free/jumbotron/index.html

**Comment by igrangel** [comment]
We need to use bootstrap since it has a lot of advantages like the _responsive design_. Therefore, I would like something like [this](https://blackrockdigital.github.io/startbootstrap-modern-business/) may be adapted with the colors palette.  

1. Bootstrap is already set in the current implementation. Please integrate the template.
2. After the integration of the template, we can update the colors.  

Waiting for updates.
